nginx-auth0/nginx.conf

```diff
@@ -34,6 +34,9 @@ http {
   lua_shared_dict userinfo 20m;
   lua_capture_error_log 32m;
 
+  # Cache 204 responses
+  proxy_cache_path /data/nginx/cache levels=1:2 keys_zone=http_204:5m max_size=1g inactive=60m use_temp_path=off;
```

Comment:
What is the plan for the non-auth0 version of the proxy?

Reply:
Good question. Need to think about it.